tests/lcs/core/field/hilitedlines.livecodescript

```diff
@@ -69,3 +69,13 @@ on TestNonContiguousSelection
    set the hilitedLines of field kFieldName to empty
    TestAssert "selectedText of a list field without selected lines", the hilitedLines of field kFieldName is empty
 end TestNonContiguousSelection
+
+on TestCanSelectTextOf65535Length
+    createField
+
+    put repeatedString("abcde",13107) into kFieldName
```

Comment:
@alexbrisan I think it the failure would because this should be:
```
put repeatedString("abcde",13107) into field kFieldName
```
Same issue below